docs/tech/junebug.md

```diff
@@ -0,0 +1,9 @@
+# Junebug
+
+[Junebug][] is an HTTP API for sending messages. It uses code from [vumi][].
```

Comment:
"sending messages" is not really the way I've thought about Junebug, but maybe that is the simplest way to put it. The intro sentence on the Junebug readthedocs is... not really easy to understand at all 😓 .

Instead of saying _"It uses code from Vumi"_ you could rather say something like "It provides an API for configuring Vumi transports"... but I'm not sure that's any more clear?